### PR TITLE
TrimArraySpacesFixer - fix moving first comment

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

**Comment by keradus** [comment]
Maybe this is what #1141 was about

File: Symfony/CS/Tests/Fixer/Symfony/TrimArraySpacesFixerTest.php [hunk 1]
```diff
@@ -329,8 +421,8 @@ public function provideFixCases()
 
             // crazy nested garbage pile #1
             array(
-                "<?php \$foo = array(/* comment \$bar = array([ ], array( 'foo' ) ) */, function(\$a = array('foo'), \$b = [/* comment [] */]) {}, array('foo' => 'bar', 'baz' => \$x[  4], 'hash' => array(1,2,3)));",
-                "<?php \$foo = array( /* comment \$bar = array([ ], array( 'foo' ) ) */, function(\$a = array( 'foo' ), \$b = [ /* comment [] */ ]) {}, array( 'foo' => 'bar', 'baz' => \$x[  4], 'hash' => array(1,2,3 )) );",
+                "<?php \$foo = array(/* comment \$bar = array([ ], array( 'foo' ) ), */ function(\$a = array('foo'), \$b = [/* comment [] */]) {}, array('foo' => 'bar', 'baz' => \$x[  4], 'hash' => array(1,2,3)));",
+                "<?php \$foo = array( /* comment \$bar = array([ ], array( 'foo' ) ), */ function(\$a = array( 'foo' ), \$b = [ /* comment [] */ ]) {}, array( 'foo' => 'bar', 'baz' => \$x[  4], 'hash' => array(1,2,3 )) );",
```
Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/TrimArraySpacesFixerTest.php]: this was a syntax before :(

**Comment by keradus** [comment]
Looks green, could anyone take a look please?

**Comment by GrahamCampbell** [comment]
:ship:

File: Symfony/CS/Tests/Fixer/Symfony/TrimArraySpacesFixerTest.php (hunk 1 above)
Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/TrimArraySpacesFixerTest.php]: ouch... ;)

**Comment by SpacePossum** [comment]
looks go to me :mountain_cableway: :+1:

**Pull request closed without merging**